go.mod

```diff
@@ -1,26 +1,114 @@
 module github.com/jetstack/google-cas-issuer
 
-go 1.16
+go 1.19
 
 require (
-	cloud.google.com/go/security v0.1.0
-	github.com/go-logr/logr v0.4.0
+	cloud.google.com/go/security v1.4.1
+	github.com/cert-manager/cert-manager v1.9.1
+	github.com/go-logr/logr v1.2.3
 	github.com/golang/protobuf v1.5.2
 	github.com/google/uuid v1.3.0
-	github.com/jetstack/cert-manager v1.5.1
 	github.com/olekukonko/tablewriter v0.0.5
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.14.0
-	github.com/spf13/cobra v1.2.1
-	github.com/spf13/viper v1.8.1
-	github.com/stretchr/testify v1.7.0
-	google.golang.org/api v0.54.0
-	google.golang.org/genproto v0.0.0-20210813162853-db860fec028c
-	google.golang.org/protobuf v1.27.1
-	k8s.io/api v0.21.3
-	k8s.io/apimachinery v0.21.3
-	k8s.io/cli-runtime v0.21.3
-	k8s.io/client-go v0.21.3
-	k8s.io/klog/v2 v2.8.0
-	sigs.k8s.io/controller-runtime v0.9.6
+	github.com/onsi/ginkgo/v2 v2.1.4
+	github.com/onsi/gomega v1.19.0
+	github.com/spf13/cobra v1.5.0
+	github.com/spf13/viper v1.12.0
+	github.com/stretchr/testify v1.8.0
+	google.golang.org/api v0.94.0
+	google.golang.org/genproto v0.0.0-20220822174746-9e6da59bd2fc
+	google.golang.org/protobuf v1.28.1
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/cli-runtime v0.24.2
+	k8s.io/client-go v0.25.0
+	k8s.io/klog/v2 v2.70.1
+	sigs.k8s.io/controller-runtime v0.12.3
```

Comment:
Note: so this jump to a version of c/r that uses leases only for leaderelection should be ok as the previous version (v0.9.6) was using configmaplease type https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.9.6/pkg/leaderelection